client/cody/package.json

```
@@ -318,78 +318,94 @@
         }
       ],
       "comments/commentThread/context": [
-				{
-					"command": "cody.comment.add",
-					"group": "inline",
+        {
+          "command": "cody.comment.add",
+          "group": "inline",
           "when": "cody.activated && commentController =~ /^cody-inline/"
-				}
-			],
+        }
+      ],
       "comments/commentThread/title": [
-				{
-					"command": "cody.comment.delete",
-					"group": "inline@1",
-					"when": "cody.activated && commentController =~ /^cody-inline/ && cody.replied && !commentThreadIsEmpty"
-				},
         {
-					"command": "cody.comment.load",
-					"group": "inline@2",
-					"when": "cody.activated && commentController =~ /^cody-inline/ && cody.reply.pending"
-				}
-			]
+          "command": "cody.comment.delete",
+          "group": "inline@1",
+          "when": "cody.activated && commentController =~ /^cody-inline/ && cody.replied && !commentThreadIsEmpty"
+        },
+        {
+          "command": "cody.comment.load",
+          "group": "inline@2",
+          "when": "cody.activated && commentController =~ /^cody-inline/ && cody.reply.pending"
+        }
+      ]
     },
     "configuration": {
       "type": "object",
-      "title": "Cody settings",
+      "title": "Cody",
       "properties": {
         "cody.enabled": {
+          "order": 1,
           "type": "boolean",
           "default": true,
           "description": "Enable Cody"
         },
         "cody.serverEndpoint": {
+          "order": 2,
           "type": "string",
           "default": "https://sourcegraph.com",
           "example": "https://example.sourcegraph.com",
           "description": "URL to the Sourcegraph instance."
         },
         "cody.codebase": {
+          "order": 3,
           "type": "string",
+          "markdownDescription": "Repo path that cody will use to gather context for the answers. Example: 'github.com/sourcegraph/sourcegraph' ",
           "example": "github.com/sourcegraph/sourcegraph"
         },
-        "cody.debug": {
-          "type": "boolean"
-        },
         "cody.useContext": {
+          "order": 4,
           "type": "string",
           "enum": [
             "embeddings",
             "keyword",
             "none",
             "blended"
           ],
-          "default": "embeddings"
+          "default": "embeddings",
+          "markdownDescription": "If embeddings for a repo are present, Cody will use them to set the context for Sourcegraph search (best scneario). If not, it will automatically fall back to keyword based search"
         },
         "cody.experimental.suggestions": {
+          "order": 5,
           "type": "boolean",
+          "markdownDescription": "Enables code suggestions while typing in the code editor window",
```

The flag has the old name but let's use the right one in the docs, wdyt?

```suggestion
          "markdownDescription": "Enables code completions while typing in the code editor window",
```